Ullaakut/disgo
# Unnecessarily bloated pull request

## Goal of this PR

* Renames this repository from `colog` to `disgo` (because it's colorful and I really want a disco gopher as the logo)
* Separates logger and prompter into separate packages
* Adds many features and improve documentation on the Prompter
	* Confirmation prompts can now prompt again until they get a valid user input
	* Confirmation prompts can now take custom choices
	* Confirmation prompts can now have no default values (it was previously mandatory to specify a default value)
	* 100% test coverage on the prompter package
* Logger:
	* Can now handle an additional error output
	* Has new `Error()`, `Errorf()` and `Errorln()` methods for writing on the error output
* Documentation improvements
	* Readme now describes the features of this project
	* Readme contains output examples
	* The `examples` folder contains more examples

## How to test it

* Run the examples and verify that it all works
* Verify that all unit tests are working
